services/dask-sidecar/src/simcore_service_dask_sidecar/computational_sidecar/core.py

```diff
@@ -75,6 +82,17 @@ async def _write_input_data(
 
                 destination_path = task_volumes.inputs_folder / file_name
 
+                need_extraction = check_need_unzipping(
```

Comment:
MINOR: Does "extraction" and "unzip" and "receivedZipData" mean the same in this context?
If so try to be consistent with names of the check function, variable and exception